### ISSUE TITLE
Benchmark SleekDB

### ISSUE THREAD

**Issue by REDAL** [rakibtg/SleekDB#214]
@Timu57 @rakibtg 
Hi,

How about making a benchmark for SleekDB, for example, how many files, volume storage, and speed can SleekDB handle easily, it will be very beneficial for newcomers, to know approximately, what they can expect from SleekDB,

I know that this will deeply depend on the hardware configuration, and but can make a benchmark based on available characteristics, I can provide a hosted account for making these tests. or you can give me the code needed to do the tests, and I ll be very happy to run it on my VPS, so we can see the potential of SleekDB

best regards

**Comment by Timu57** [comment]
Hi @REDAL 
Benchmarks sound nice, feel free to make some :)

**Comment by REDAL** [comment]
@Timu57 
@johnnestebann already did one, and I am very happy with the results, I actually like it is my library :) just loved it

**Comment by johnnestebann** [comment]
@REDAL are you kidding me? I never see this post when I did my benchmarks test! it is incredible that you proposed the benchmark and 3hr later I was testing the package for my own haha I'm surprised!

**Comment by REDAL** [comment]
@johnnestebann

it's not important who proposed first my friend, the idea, os to make it better ;)

and I appreciate your work really, and I did make a request, if you wish of course ;)

**Comment by johnnestebann** [comment]
hahaha no no @REDAL I'm just saying it's incredible the coincidence of thinking the same thing almost at the same time, I'm just amazed

**Comment by REDAL** [comment]
@johnnestebann  

I agree brother, and I've understood you ;), like they said, the spirits gather in their own circles.

**Comment by Timu57** [comment]
@REDAL 

You wrote that you have benchmark results. Would be awesome if you could share them with us.
Then we can show them on the documentation page.

**Comment by redaloui** [comment]
@Timu57 
You miss understood me, the first time I did siggested to do one, then, it was done by @johnnestebann .

After, I suggested to add it to the documentation page, so new comers undersrand the potential of SleekDB and adopt it quickly 💙

**Comment by Awilum** [comment]
Any benchmarks for SleekDB exists?
